Keas.Core/Domain/KeyXSpace.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Keas.Core.Domain
+{
+    public class KeyXSpace
+    {
+        [Key]
+        public int Id { get; set; }
+
+        public Key Key { get; set; }
+        public int KeyId { get; set; }
+
+        public Space Space { get; set; }
+        public int? SpaceId { get; set; }
```

Comment:
@jcubbage why is this nullable?  Doesn't seem like there should be the possibility of having an entry in this table w/o both a keyId and spaceId.